base/common/src/org/dogtagpki/tps/apdu/InitializeUpdateAPDU.java

```diff
@@ -56,6 +56,8 @@ public TPSBuffer getEncoding()
         theData.add(p2);
         theData.add((byte) data.size());
         theData.add(data);
+        // Add Le byte 
+        theData.add((byte) 0x00);
```

Comment:
same here.  he old cards work fine with this change?

Reply:
Yes, this is the  "le" trailing byte issue. Older cards assume it but some cards insist that it is present. So the older cards are ok with it being present.